eth/v1alpha1/validator.proto

```diff
@@ -174,6 +175,7 @@ service BeaconNodeValidator {
     rpc GetAttestationData(AttestationDataRequest) returns (AttestationData) {
         option (google.api.http) = {
             get: "/eth/v1alpha1/validator/attestation"
+            body: "*"
```

Comment:
This causes build to fail

Reply:
Oh, this is a `get`. Don't know how I missed that